README.md

```diff
@@ -0,0 +1,12 @@
+
+Getting Started
+
```

Comment:
Almost. I think it might be simpler as this:

```bash
cmake .
make
./run_trials.sh
```

Reply:
I like the build directory because it keeps all the build artifacts contained. I agree with the simpler way and no symlink